### ISSUE TITLE
[Workshop] Inverted cursor color

### ISSUE THREAD

**Issue by Metacinnabar** [Metacinnabar/OptimizedCursor#5]
I really like this mod, it made me realize how choppy terraria's cursor really is. However I am having one small issue. Seemingly at random my cursor will swap from just being a suped up version of my default cursor to some kind of odd adaptive inverted color mode? I wouldn't mind this usually, but it also locks my cursor visually into looking like the squashed style cursor. I can still swap the styles/functions but the cursor dosen't reflect the change. Is there any way to make this not happen, like an unlisted settings menu or something?

**Comment by Metacinnabar** [comment]
[Workshop]
I also seem to get the inverted cursor color issue when messing with the border color sliders at the main menu. It fixes itself after a game restart. I do use a custom cursor resource pack, so not sure if this somehow causes a conflict somewhere, or its a bug with certain color settings.

**Comment by Metacinnabar** [comment]
I basically cosign 100% of what @when said.

I thought it may have been an issue with that Rainbow Cursor accessory, but I unequipped it and it still happened even after I closed my game, verified integrity of files, and restarted.

I'm not sure if the problem is tied to messing with the cursor color settings in the menu because I never did that while testing the mod and the cursor still eventually inverted.

Of note: The only mods off the top of my head that I think could be messing with the cursor is Better Game UI and Terraria Overhaul but it could be something else, or nothing at all.
